site/src/pages/components/radio.research.mdx

```diff
@@ -38,35 +39,35 @@ After the research there are still [discussions to be made](#discussion), where
 
 ## Anatomy
 
-<Anatomy component="Radio Button" />
+<Anatomy component="Radio" />
 
 ## Concepts
 
-<Concepts client:load component="Radio Button" />
+<Concepts client:load component="Radio" />
 
 ## Discussion
 
-At a point during the research, the scope of this study os shifted from the “radio button” to the “radio button group”, because there is no particular strong use case for a standalone “radio button”, and also because this use case is against the W3 Specification, the actual component to be considered is the “radio button group”, being composed of two or more radio buttons.
+At a point during the research, the scope of this study os shifted from the “radio” to the “radio group”, because there is no particular strong use case for a standalone “radio”, and also because this use case is against the W3 Specification, the actual component to be considered is the “radio group”, being composed of two or more radios.
```

Comment:
remove _os_ as "the scope of this study shifted" is sufficient

> there is no strong usecase for a radio

I disagree with this stance as I've heard the same statement for checkbox but can make an argument for it